modules/spotxBidAdapter.md

```diff
@@ -0,0 +1,148 @@
+# Overview
+
+```
+Module Name: SpotX Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: teameighties@spotx.tv
+```
+
+# Description
+
+Connect to SpotX for bids.
+
+This adapter requires setup and approval from the SpotX team.
+
+# Test Parameters - Use case #1 - outstream with default rendering options
+```
+    var adUnits = [{
+        code: 'something',
+        mediaTypes: {
+            video: {
+                context: 'outstream', // 'instream' or 'outstream'
+                playerSize: [640, 480]
+            }
+        },
+        bids: [{
+            bidder: 'spotx',
+            params: {
+                channel_id: 79391,
+                ad_unit: 'outstream',
+                outstream_options: { // Needed for the default outstream renderer - fields video_slot/content_width/content_height are mandatory
+                    video_slot: 'video1',
+                    content_width: 300,
```

Comment:
`content-width` / `content_height` don't match the `playerSize` attribute. Is this okay?

Reply:
It's fine. playerSize will be the size sent for the adcall and will be used for demand purposes. content_width and content_height will be used for the outstream player renderer.